CHANGELOG.md

```diff
@@ -8,11 +8,9 @@ Please see the [upgrade guide](UPGRADING.md) for details of all the changes and
 
 ### Bug fixes
   
-* Stop scene changes overiding context when manually set
-  [#255](https://github.com/bugsnag/bugsnag-unity/pull/255)
+* Correct android session start times
+  [#291](https://github.com/bugsnag/bugsnag-unity/pull/291)
 
-* Dont Destroy TimingTrackerObject, so it persists across scenes
-  [#239](https://github.com/bugsnag/bugsnag-unity/pull/239)
```

Comment:
Is this a conflict of some kind? It looks like the removed lines are from 5.0.0, so maybe a new section is needed for 5.1.0?

Reply:
yeah i'm going to fix the changelog once these 3 prs are in if that's ok. it's becaues it's based on an unreleased branch i think